### PR TITLE
MC-15368 Adds billing settings doc

### PR DESCRIPTION
**Pull request by m-orfanos**
### Fixes [MC-15368](https://cloud-ops.atlassian.net/browse/MC-15368)

New billing settings section

image

#### Related PRs

https://github.com/cloudops/cloudmc-core/pull/1386
https://github.com/cloudops/cloudmc-ui/pull/1710

### CODE REVIEW

File: source/includes/administration/_billing_settings.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+#### Find billing settings associated to an organization
+
+`GET /reseller/settings/billing/find?organizationId=:id`
+
+Retrieve the billing settings associated to an organization. If the `organizationId` is omitted, the authenticated user's organization will be used.
+
+```shell
+# Retrieve the billing settings
+curl "https://cloudmc_endpoint/api/v1/reseller/settings/billing/find?organizationId=10572c3d-16e5-450f-8af8-a01e50dc52d4" \
```
Comment (razine-bensari) [inline source/includes/administration/_billing_settings.md]: isnt supposed to be:
```suggestion
curl "https://cloudmc_endpoint/api/v1/rest/reseller/settings/billing/find?organizationId=10572c3d-16e5-450f-8af8-a01e50dc52d4" \
```

Comment (razine-bensari) [reply source/includes/administration/_billing_settings.md]: I am not sure that is why I am asking, we usually use the `rest` path for the api but I might be wrong I havent tested this endpoint yet.

But if yes, then I think we may need to modify the ones below

Comment (razine-bensari) [reply source/includes/administration/_billing_settings.md]: If not please disregard my comment or resolve it

Comment (m-orfanos) [reply source/includes/administration/_billing_settings.md]: yeah, the `v1` needs to be dropped in favour of `rest`

**Review by razine-bensari — APPROVED** [review]
👍🏼 
Just added a comment but I may be wrong

File: source/includes/administration/_billing_settings.md (hunk 1 above)
Comment (m-orfanos) [reply source/includes/administration/_billing_settings.md]: done aa2d00770506d18f625dbda219bac3f6bf929b6b

File: source/includes/administration/_billing_settings.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+`version`<br/>*integer* | The billing settings version.
+`daysBeforeAutoApproval`<br/>*integer* | The number of days to review a draft invoice before it is automatically issued to the customer.
+`daysBeforeAutoPayment`<br/>*integer* | The number of days after the invoice has been issued before the customer's credit card is automatically charged.
+
+<!-------------------- CREATE BILLING SETTINGS -------------------->
+#### Create billing setting
+
+`POST /reseller/settings/billing`
+
+Create a new billing settings.
```
Comment (tylerrwatsonn) [inline source/includes/administration/_billing_settings.md]:
```suggestion
Create new billing settings.
```

File: source/includes/administration/_billing_settings.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The configured billing settings' id.
+`organization.id`<br/>*UUID* | The organization id that the billing settings are linked to. It cannot be changed.
+`version`<br/>*integer* | The billing settings version.
+`daysBeforeAutoApproval`<br/>*integer* | The number of days to review a draft invoice before it is automatically issued to the customer.
+`daysBeforeAutoPayment`<br/>*integer* | The number of days after the invoice has been issued before the customer's credit card is automatically charged.
+
+<!-------------------- CREATE BILLING SETTINGS -------------------->
+#### Create billing setting
```
Comment (tylerrwatsonn) [inline source/includes/administration/_billing_settings.md]: why is setting singular here?

File: source/includes/administration/_billing_settings.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+{
+  "organization": {
+    "id": "10572c3d-16e5-450f-8af8-a01e50dc52d4"
+  },
+  "daysBeforeAutoApproval": 3,
+  "daysBeforeAutoPayment": 5,
+}
+```
+
+> The above command return JSON structured like this:
```
Comment (tylerrwatsonn) [inline source/includes/administration/_billing_settings.md]:
```suggestion
> The above command returns JSON structured like this:
```

File: source/includes/administration/_billing_settings.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+    "id": "10572c3d-16e5-450f-8af8-a01e50dc52d4"
+  },
+  "id": "d785ffcb-9b03-478d-a49b-52a2ccedf1b8",
+  "version": 1,
+  "daysBeforeAutoApproval": 3,
+  "daysBeforeAutoPayment": 5,
+}
+```
+
+> The above command return JSON structured like this:
```
Comment (tylerrwatsonn) [inline source/includes/administration/_billing_settings.md]:
```suggestion
> The above command returns JSON structured like this:
```

**Review by tylerrwatsonn — APPROVED** [review]
Minor comments but looks good otherwise!

File: source/includes/administration/_billing_settings.md (hunk 3 above)
Comment (m-orfanos) [inline source/includes/administration/_billing_settings.md]:
```suggestion
#### Create billing settings
```

Comment (m-orfanos) [reply source/includes/administration/_billing_settings.md]: fixed

**Pull request merged**